CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ project(exiv2
 include(cmake/mainSetup.cmake  REQUIRED)
 include(CTest)
 
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
+include (gcovr)
```

Comment:
[minor] Although this is correct, we are trying in this project to not modify the `CMAKE_MODULE_PATH` internally so that we can clearly differentiate between the CMake provided modules and the ones provided by us. 

Please remove line 11 and change line 12 to:

```
include(cmake/gcovr.cmake REQUIRED)
```